.travis.yml

```diff
@@ -0,0 +1,12 @@
+dist: xenial
+language: python
+python:
+  - "2.7"
```

Comment:
I don't know the real purpose of the project but since it is a brand new project I think that worth trying to do Python 3.6+. Python 3 brings a lot of cool stuff that you can explore that I really think worth giving it a try.

If you need help converting to Python 3 please let me know and I can try helping.

Also this comment is not a blocker, just feed for thought :)

Reply:
Thanks @elyezer for looking into it. I earlier thought of having it on python3 but pytest-ansible (https://github.com/jlaska/pytest-ansible) on which this project is based on is not compatible with python3. Once py3 support added to pytest-ansible, we will move to py3.